test/functional/test_framework/test_framework.py

```diff
@@ -1594,7 +1583,6 @@ def dynamically_add_masternode(self, evo=False, rnd=None, should_be_rejected=Fal
         mn_idx = len(self.nodes)
 
         node_p2p_port = p2p_port(mn_idx)
```

Comment:
could move `node_p2p_port` calculations into `dynamically_prepare_masternode` and drop it from params